### ISSUE TITLE
BUG - sev-2 - All - Appointments: cancellation failure snackbar is not appearing

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#8706]
## What happened?

When attempting to cancel an appointment, a snackbar should appear that tells the user whether or not the appointment was successfully cancelled ([this test case](https://dsvavsp.testrail.io/index.php?/cases/view/18043&group_by=cases:section_id&group_order=desc&display_deleted_cases=0&group_id=3388), which I am assuming is still the desired behavior since there isn't any new case for appt cancel behavior in there replacing it, nor has it been archived).

I'm failing to cancel appointments, and it's not showing the appointment cancellation failure snackbar. See [video](https://drive.google.com/file/d/1DJUmFsmYzGWTKEIsE4XY3kQdA6xVEZiV/view?usp=sharing). (Also I apologize for the looooooooong wait in the video, that's going to be written up as another ticket. (edit - reopened #4762)).

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Desired behavior

Snackbar should appear

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

sev-2 for the accessibility concerns/there isn't a workaround

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
